kkanbu/board/views.py

```diff
@@ -30,17 +31,9 @@ class AbstractPostViewSet(ModelViewSet):
     """Post Model을 기반으로 한 객체를 다루는 필요한 공통 속성만 뽑아낸 ViewSet"""
 
     permission_classes = [IsOwnerOrReadOnly]
+    filter_backends = [filters.SearchFilter, PostOrderingFilter, TagFilter]
```

Comment:
프론트엔드나 제3자라면 주로 코드 보다는 api 문서를 보지 않을까 싶어서 코드 좀 찾아왔어!
여기 class를 swagger 문서로 ordering이나 search가 parameter로 사용된다는 걸 알려주는 게 더 나을듯!
```suggestion
    @extend_schema(
        # extra parameters added to the schema
        parameters=[
            OpenApiParameter(name='ordering', 
              description="Ordering by field is available. Fields are 'created', 'hit'",
              location=OpenApiParameter.QUERY,
               required=False, type=str),
            OpenApiParameter(
                name='ordering',
                type=str,
                location=OpenApiParameter.QUERY,
                description="Search on  fields. Fields are 'title', 'content'",
                ])
```

Reply:
잘 작동하지는 않네... 아직 어떻게 사용하는지 잘 모르겠다